### DISCUSSION TITLE
Using symbol from loaded library to trigger tracing?

### DISCUSSION THREAD

**Discussion by smarr** [janestreet/magic-trace#206] [General]
Is there any way to trigger tracing based on a symbol that's not available in the given binary, but only becomes available after loading a library?

I am attempting to trace a project that is split into a launcher and a library with the actual functionality, and magic trace doesn't find the relevant symbol.

Thank you

**Comment by Xyene** [comment]
See https://github.com/janestreet/magic-trace/discussions/189.

Patches very welcome :)
